test/const.test.js

```diff
@@ -238,7 +238,7 @@ test('schema with const and null as type', (t) => {
     foo: null
   })
 
-  t.equal(output, '{"foo":null}')
+  t.equal(output, '{"foo":"baz"}')
```

Comment:
`JSON.stringify({foo: null })`  is `{"foo":null}`

So how can it be that stringifying `{ foo: null }` results in `{"foo": "baz} `

Reply:
FJS output should be equal to JSON.stringify only when you pass data that match the data schema. You shouldn't pass `null` as a data input for schema { "type": ["string", "null"], "const": "abc" }, because null doesn't match the schema.

Reply:
We want this to work:
```js
const schema = { /*some json schema draft 7*/ }
const data = { /*data that match to schema or can be coerced according to FJS rules */ }

const json = fjs(data)
const parsedData = JSON.parse(json)

validate(schema, data) // should be always true
```

And if we serialize `null` as `null`, 
```js
validate({ "type": ["string", "null"], "const": "abc" }, null)
```
will return false.

Reply:
For the same reason, if you pass `null` to the serializer with schema `{ "type": "number" }`, you will get `0`, instead of `null`.